sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.3.1 (2021-04-05)
+
```

Comment:
I know it is not related, but I was hoping on piggy backing on your PR rather than create a new one for the hot fix branch :)

```suggestion
- With this release, we drop support for Node.js versions that have reached end of life like Node.js 8. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
```

Reply:
Oh nice!! Thank you!

Reply:
Done!